circuits/verify_eddsamimc.circom

```diff
@@ -1,5 +1,5 @@
-include "../circomlib/circuits/eddsamimc.circom";
-include "../circomlib/circuits/mimc.circom";
+include "../node_modules/circomlib/circuits/eddsamimc.circom";
```

Comment:
Is this hot handled directly in "../node_modules/circomlib/circuits/eddsamimc.circom";

Reply:
Not for messages of arbitrary length